src/rp2_common/wifi_settings_connect/include/wifi_settings/wifi_settings_configuration.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2025 Jack Whitham
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * This header file contains default values for timeouts,
+ * addresses and limits within the pico-wifi-settings library.
+ */
+
+#ifndef WIFI_SETTINGS_CONFIGURATION_H
+#define WIFI_SETTINGS_CONFIGURATION_H
+
+
+#if defined(FLASH_ADDRESS_OF_WIFI_SETTINGS_FILE)
+// Flash address of wifi-settings file already defined
+#elif PICO_RP2040
+// Flash address of wifi-settings file on Pico 1 W
+#define FLASH_ADDRESS_OF_WIFI_SETTINGS_FILE         0x001ff000
```

Comment:
Note that 3rd-party RP2040 and RP2350 boards may have different size Flash chips, so _perhaps_ this should be calculated as an offset from the end of Flash, rather than being a hardcoded address? (which I image would be pretty inconvenient if you're e.g. using a 16MB Flash chip!)
See e.g. https://github.com/raspberrypi/pico-sdk/pull/2412 (and I guess you'll want to check that your Flash-usage doesn't conflict with the BlueTooth flash-usage)